docs/source/intro-clx-loda-anomaly-detection.ipynb

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
<p>change Loda to LODA for consistency.</p><p> </p><p>Can you include a link to a LODA research paper for reference?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Reference is at the bottom of the NB.</p>

Reply:
I will add them